### ISSUE TITLE
Status for buses that clearly aren't coming

### ISSUE THREAD

**Issue by cibyr** [OneBusAway/onebusaway#23]
[This](https://theneedling.com/2023/03/06/onebusaway-finally-adds-never-notification-for-when-your-bus-has-fallen-into-the-void/), but unironically.

**Comment by Youssef-Beltagy** [comment]
Yes, that would be very helpful. Right now, I see cases where the bus is stuck in `scheduled` but never comes.
